SingularityUI/app/components/common/table/UITable.jsx

```diff
@@ -0,0 +1,325 @@
+import React, { Component, PropTypes } from 'react';
+import Waypoint from 'react-waypoint';
+import classNames from 'classnames';
+import _ from 'underscore';
```

Comment:
I like being explicit with my imports, but if you think there's some negative effects to this, I can remove it.